test/routes/test_configRoute.ts

```
@@ -0,0 +1,184 @@
+    path: string;
+    stack: Array<{ method: string; handle: Handler }>;
+  };
+}
+
+interface RouterInternals {
+  stack: StackFrame[];
+}
+
+function extractRouteHandler(
+  mod: RouteModule,
+  path: string,
```

ESLint

[error] 'path' is already declared in the upper scope on line 19 column 8.

(no-shadow)